client/src/components/Button/Button.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import "./Button.css";
+
+const Button = props => {
+  Button.defaultProps = {
+    color: "color",
+    size: "size"
+  };
```

Comment:
So cool you're using `defaultProps`! Can you move this out of the function definition though?